editor-components/modal/notification-modal1/notificationmodal1.module.scss

```diff
@@ -0,0 +1,197 @@
+.container {
+  width: 100%;
+
+  .max-content {
+    max-width: var(--composer-content-width);
+    margin: auto;
+    width: var(--composer-content-width);
```

Comment:
use screen width instead of --composer-content-width. cause when the --composer-content-width reduces the content width also reduces.